pkg/controller/connectivitycheck/connectivity_check_controller.go

```diff
@@ -58,7 +58,6 @@ func NewNetworkConnectivityCheckController(
 			operatorcontrolplaneClient,
 			apiextensionsClient,
 			apiextensionsInformers,
-			configInformers,
```

Comment:
hm... how come?

Reply:
fixed. This was not required, it was a mixup after removing some of the replace in the go.mod